operations/security_and_authentication/Keycloak_Service_Recovery.md

```diff
@@ -0,0 +1,138 @@
+# Keycloak Service Recovery
+
+The following covers redeploying the Keycloak service and restoring the data.
+
+## Prerequisites
+
+The system is fully installed and has transitioned off of the LiveCD.
+
+All activities required for site maintenance are complete.
+
+A backup or export of the data already exists.
+
+The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
```

Comment:
Added CLI prerequisite because I see some Cray CLI commands in the procedure.

```suggestion
- The system is fully installed and has transitioned off of the LiveCD.
- All activities required for site maintenance are complete.
- A backup or export of the data already exists.
- The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
- The Cray CLI has been configured on the node where the procedure is being performed. See [Configure the Cray CLI](../configure_cray_cli.md).
```